neighbors/models.py

```diff
@@ -0,0 +1,53 @@
+from django.conf import settings
+from django.contrib.gis.db import models as geo_models
+from django.db import models
+
+
+# We may want this data in fixtures or migrations to initially seed the DB
+ROLES = [
+    # Volunteer
+    # Requester
+    # Moderator
+]
+
+
+class Role(models.Model):
+    name = models.CharField(
+        max_length=50
+    )  # It seems like in theory someone could switch between roles -- volunteer but eventually have to self-quarantine or vice-versa
+
+
+class NeighborProfile(geo_models.Model):
```

Comment:
Not a technical critique, but I like that we're calling them 'neighbors' not 'people' or whatever. Nice touch.